examples/web-wagmi/src/profiles/UseFollowAndUnfollow.tsx

```diff
@@ -10,13 +15,19 @@ type ProfileFollowProps = {
 };
 
 function FollowButton({ profile }: ProfileFollowProps) {
-  const { follow, isPending } = useFollow({ profile });
+  const { follow, isPending: isFollowing } = useFollow({ profile });
+  const { unfollow, isPending: isUnfollowing } = useUnfollow({ profile });
 
-  if (profile.isFollowedByMe || profile.isOptimisticFollowedByMe) return <p>Following</p>;
+  if (profile.isFollowedByMe || profile.isOptimisticFollowedByMe)
+    return (
+      <button onClick={unfollow} disabled={isUnfollowing || profile.isOptimisticFollowedByMe}>
```

Comment:
is that by design that `Unfollow` button is in `disabled` state until you refrersh the page?
That's caused by `useUnfollowController` being `true` even after we already updated `isFollowedByMe` to be `true`. 
Can we revert to `false` `isFollowedByMe` in `FollowResponser` `commit` phase to unblock unfollowing?